src/client/lobby/lobby.js

```diff
@@ -7,6 +7,7 @@ import RoomList from './room-list';
 import RoomCreator from './room-creator';
 
 import { refreshLobby } from '../actions';
+import { refreshRooms } from '../actions';
 
```

Comment:
Can be squashed to one neat wee line
`import { refreshLobby, refreshRooms } from '../actions';`